app/src/main/java/com/babylon/wallet/android/utils/DateExtensions.kt

```diff
@@ -55,14 +55,21 @@ fun Instant.dayMonthDateShort(): String {
 fun Instant.dayMonthDateFull(): String {
     val zoneId = ZoneId.systemDefault()
     val currentYear = Instant.now().atZone(zoneId).year
+    val currentDay = Instant.now().atZone(zoneId).dayOfYear
     val instantYear = atZone(zoneId).year
+    val instantDay = atZone(zoneId).dayOfYear
     val format = if (currentYear == instantYear) {
         LAST_USED_DATE_FORMAT_THIS_YEAR
     } else {
         LAST_USED_DATE_FORMAT
     }
+    val prefix = when {
+        currentDay == instantDay -> "Today, "
```

Comment:
⚠️ I would change all this method to be used in `@Composable` only. So here you need to get the today and yesterday localised resources and not hardcoded